LightBlue.MultiHost/Runners/RoleRunner.cs

```diff
@@ -46,6 +46,30 @@ public RoleRunner(Role role)
 
         public void Start()
         {
+            if (_role.RoleName == "DotNetCore")
```

Comment:
can we apply a bit of DRY here, perhaps something like;
```
var role =
    _role.RoleName == "DotNetCore" ? RunnerFactory.CreateDotNetCoreRunner(_role) :
    _role.RoleName == "Console" ? RunnerFactory.CreateConsoleRunner(_role) : 
    ...

_resources.Add(role);
role.Start();
```

Reply:
In fact, perhaps this decision logic is actually an internal concern of the `RunnerFactory`? ie
`var role = RunnerFactory.CreateRunner(_role)` ?

Reply:
Agree encapsulating that logic inside the factory is a better design but I wasn't really looking to refactor just following what's already in place (CreateForWebsite, CreateForWorkerRole etc).